xml/System.Text.Encodings.Web/HtmlEncoder.xml

```diff
@@ -103,9 +109,9 @@
         <ReturnType>System.Text.Encodings.Web.HtmlEncoder</ReturnType>
       </ReturnValue>
       <Docs>
-        <summary>To be added.</summary>
-        <value>To be added.</value>
-        <remarks>To be added.</remarks>
+        <summary>Gets a default built-in instance of the <see cref="T:System.Text.Encodings.Web.HtmlEncoder"/> class.</summary> 
+        <value>A default built-in instance of the <see cref="T:System.Text.Encodings.Web.HtmlEncoder"/> class.</value>
```

Comment:
It seems redundant to say it returns the default built-in instance. It's OK to say it returns the default instance, but in that case we should specify in remarks what the defaults are (in the source, it's indicated by the internal DefaultHtmlEncoder class); otherwise, built-in instance is best.

This also is true of the other encoder Default properties.